src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -8,13 +8,64 @@ categories:
   - mobile
 ---
 
-The `sentry-compose-android` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) support for the following interactions:
-* User interactions like clicks, swipes and scroll gestures
-* Jetpack Compose navigation
+The `sentry-compose-android` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) support in the following areas:
+* `@Composable` performance metrics
+* Transaction and breadcrumb support for User interactions (clicks, swipes and scroll gestures)
+* Transaction and breadcrumb support for Navigation events
+
+## Performance Metrics
+_(New in version 6.16.0)_
+
+With Jetpack Compose performance metrics the Sentry Android SDK can automatically measure the initial `composition` and `rendering` time as performance spans of your `@Composable` UI elements.
+
+<Note>
+
+This feature relies on Automatic Activity Instrumentation, check the <PlatformLink to="/performance/instrumentation/automatic-instrumentation/">performance documentation</PlatformLink> on how to get started first.
+
+</Note>
+
+Once enabled you'll see two spans inside your transactions:
+* `ui.compose.composition`, describing the time it took to execute your `@Composable`
+* `ui.compose.rendering`, describing the time it took to render your `@Composable` onto the canvas
+
+![Jetpack Compose Performance Metrics](jetpack-compose-performance-metrics.png)
+
+In order to use it, you need to decorate your `@Composable` body with the `SentryTraced` function.
```

Comment:
```suggestion
Once performance metrics have been enabled, you'll see two spans inside your transactions:
* `ui.compose.composition`, describing the time it took to execute your `@Composable`
* `ui.compose.rendering`, describing the time it took to render your `@Composable` to the canvas

![Jetpack Compose Performance Metrics](jetpack-compose-performance-metrics.png)

In order to use it, you need to decorate your `@Composable` body with the `SentryTraced` function.
```
Is "decorate" the right word here? Haven't seen it used in this context before :)

Reply:
Good point, I think _wrap_ is a more common term, let's use it instead.